### PR TITLE
Update quick start guide

### PR DESCRIPTION
**Pull request by ghost**
Simpler instructions with Cultivation. Thanks to Nazrin for writing.

### CODE REVIEW

**Comment by reflexit** [comment]
Should we keep both Cultivation method (recommended) and manual method (for advanced users)? Because not every user downloads Cultivation, some (including me) prefer configuring things manually.

**Comment by ghost** [comment]
Sure, I don't see why not. Feel free to propose something for the manual method and I'll put it there + add you as a co-author to the commit.

**Comment by reflexit** [comment]
The manual method is basically the `Quick setup guide` part in the current README. Perhaps we can clean it up a little bit and remove some obsolete information.

As a reference, my setup steps are:
1. Install java, mongodb, and a proxy daemon (I use mitmproxy).
2. Get `grasscutter.jar` and `resources` folder.
3. Start mongodb service if not already. On Windows the command is `net start mongodb`.
4. Run Grasscutter with `java -jar grasscutter.jar`.
5. Run proxy daemon and set system network proxy (see `Connecting with the client` part).
6. Install a certain client patch. Run the game **as administrator** (very important).

I've been following these steps since 3.3, and it still works now.

**Comment by KingRainbow44** [comment]
the word `Genshin` is a big no-no

File: README.md [hunk 1]
```diff
@@ -22,51 +22,21 @@
...
+• Get Java 17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
+• Get MongoDB Community Server: https://www.mongodb.com/try/download/community
+• Get Genshin version 3.7 (Genshin 3.7 client can be found here if you don't have it): https://github.com/MAnggiarMustofa/GI-Download-Library/blob/main/GenshinImpact/Client/3.7.0.md)
+
+• Download the latest Cultivation version. Install the .msi: https://github.com/Grasscutters/Cultivation/releases/latest
+• After opening Culivation (as admin), press the download button in the upper right corner. 
+• Click Download All-in-One
+• Click the gear in the upper right corner
+• Set the game Install path to where Genshin Impact\Genshin Impact game\GenshinImpact.exe is.
+• Set the Custom Java Path to C:\Program Files\Java\jdk-17\bin\java.exe
```
Comment (KingRainbow44) [inline README.md]:
```suggestion
• Set the Custom Java Path to `C:\Program Files\Java\jdk-17\bin\java.exe`
```

File: README.md [hunk 2]
```diff
@@ -22,51 +22,21 @@
...
-    - Fiddler Classic: Run Fiddler Classic, turn on `Decrypt HTTPS traffic` in (Tools -> Options -> HTTPS) and change the default port in (Tools -> Options -> Connections) to anything other than `8888`, load [this script](https://github.com/Grasscutters/Grasscutter/wiki/Resources#fiddler-classic-jscript) (copy and paste the script in the `FiddlerScript` tab) and click the `Save Script` button.
-
-    - [Hosts file](https://github.com/Grasscutters/Grasscutter/wiki/Resources#hosts-file)
-
-2. Set network proxy to `127.0.0.1:8080` or the proxy port you specified.
-
-- For mitmproxy: After setting up the network proxy and installing the certificate, check http://mitm.it/ if traffic is passing through mitmproxy.
-
-**You can also use `start.cmd` to start servers and proxy daemons automatically, but you have to set up `JAVA_HOME` environment and configure the `start_config.cmd` file.**
+• Get Java 17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
```
Comment (KingRainbow44) [inline README.md:27]:
```suggestion
### Quick Start (automatic)

• Get Java 17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
```

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -22,51 +22,21 @@
...
+• Get MongoDB Community Server: https://www.mongodb.com/try/download/community
+• Get Genshin version 3.7 (Genshin 3.7 client can be found here if you don't have it): https://github.com/MAnggiarMustofa/GI-Download-Library/blob/main/GenshinImpact/Client/3.7.0.md)
+
+• Download the latest Cultivation version. Install the .msi: https://github.com/Grasscutters/Cultivation/releases/latest
+• After opening Culivation (as admin), press the download button in the upper right corner. 
+• Click Download All-in-One
```
Comment (KingRainbow44) [inline README.md]:
```suggestion
• Click `Download All-in-One`
```

File: README.md [hunk 4]
```diff
@@ -22,51 +22,21 @@
...
+• Click the gear in the upper right corner
+• Set the game Install path to where Genshin Impact\Genshin Impact game\GenshinImpact.exe is.
```
Comment (KingRainbow44) [inline README.md]:
```suggestion
• Set the game Install path to where your game is located.
```

File: README.md [hunk 5]
```diff
@@ -22,51 +22,21 @@
...
-2. Set network proxy to `127.0.0.1:8080` or the proxy port you specified.
-
-- For mitmproxy: After setting up the network proxy and installing the certificate, check http://mitm.it/ if traffic is passing through mitmproxy.
-
-**You can also use `start.cmd` to start servers and proxy daemons automatically, but you have to set up `JAVA_HOME` environment and configure the `start_config.cmd` file.**
+• Get Java 17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
+• Get MongoDB Community Server: https://www.mongodb.com/try/download/community
+• Get Genshin version 3.7 (Genshin 3.7 client can be found here if you don't have it): https://github.com/MAnggiarMustofa/GI-Download-Library/blob/main/GenshinImpact/Client/3.7.0.md)
+
+• Download the latest Cultivation version. Install the .msi: https://github.com/Grasscutters/Cultivation/releases/latest
```
Comment (KingRainbow44) [inline README.md]:
```suggestion
• Download the [latest Cultivation version](https://github.com/Grasscutters/Cultivation/releases/latest). Use the `.msi` installer.
```

File: README.md [hunk 6]
```diff
@@ -22,51 +22,21 @@
...
-    - [Hosts file](https://github.com/Grasscutters/Grasscutter/wiki/Resources#hosts-file)
-
-2. Set network proxy to `127.0.0.1:8080` or the proxy port you specified.
-
-- For mitmproxy: After setting up the network proxy and installing the certificate, check http://mitm.it/ if traffic is passing through mitmproxy.
-
-**You can also use `start.cmd` to start servers and proxy daemons automatically, but you have to set up `JAVA_HOME` environment and configure the `start_config.cmd` file.**
+• Get Java 17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
+• Get MongoDB Community Server: https://www.mongodb.com/try/download/community
+• Get Genshin version 3.7 (Genshin 3.7 client can be found here if you don't have it): https://github.com/MAnggiarMustofa/GI-Download-Library/blob/main/GenshinImpact/Client/3.7.0.md)
```
Comment (KingRainbow44) [inline README.md]:
```suggestion
• Get game version REL3.7 (3.7 client can be found here if you don't have it): https://github.com/MAnggiarMustofa/GI-Download-Library/blob/main/GenshinImpact/Client/3.7.0.md)
```

File: README.md [hunk 7]
```diff
@@ -22,51 +22,21 @@
...
-
-    - [Hosts file](https://github.com/Grasscutters/Grasscutter/wiki/Resources#hosts-file)
-
-2. Set network proxy to `127.0.0.1:8080` or the proxy port you specified.
-
-- For mitmproxy: After setting up the network proxy and installing the certificate, check http://mitm.it/ if traffic is passing through mitmproxy.
-
-**You can also use `start.cmd` to start servers and proxy daemons automatically, but you have to set up `JAVA_HOME` environment and configure the `start_config.cmd` file.**
+• Get Java 17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
+• Get MongoDB Community Server: https://www.mongodb.com/try/download/community
```
Comment (KingRainbow44) [inline README.md]:
```suggestion
• Get [MongoDB Community Server](https://www.mongodb.com/try/download/community)
```

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
> the word `Genshin` is a big no-no

Uh, why? 

Just mentioning the game by name doesn't infringe any [intellectual "property"](https://www.gnu.org/philosophy/not-ipr.html) rights. It seems simpler to name it.

Also, Genshin is [a Buddhist monk](https://en.wikipedia.org/wiki/Genshin), not a game.

**Review by KingRainbow44 — APPROVED** [review]

**Pull request merged**